engineering/ember.md

```diff
@@ -25,6 +26,17 @@ fullName: Ember.computed('user.firstName', 'user.lastName', {
 })
 ```
 
+## Templates
+
+### Don't use partials
+
+Use components instead of partials.
```

Comment:
You _could_ do a "good" / "bad" example. But honestly this is probably sufficient.